### PR TITLE
More resources

### LINKED ISSUE #51: Add youtube channel of rdm4mic
https://www.youtube.com/@RDM4mic

### LINKED ISSUE #53: Add Sreeni's video about meta data
https://youtu.be/A4po9z61TME?si=__1SuAICM8TlJGF2

### LINKED ISSUE #55: Add Josh's keynote lecutre
https://zenodo.org/records/10008465

### LINKED ISSUE #60: Add "Effectively Communicating Bioimage Analysis" slides?
If a list of the slides are available it would be nice to add them to the list.

I only found this one for now:
- "My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud": https://zenodo.org/records/10679054

### LINKED ISSUE #61: Add QM Course Lectures on Bio-Image Analysis with napari 2024
https://github.com/zoccoler/QM_Course_Bio_Image_Analysis_with_napari_2024

### LINKED ISSUE #62: Add Plants & Python: A series of lessons in coding, plant biology, computation, and bioinformatics
https://plantsandpython.github.io/PlantsAndPython/00_Opening_page.html
https://academic.oup.com/plcell/article/34/7/e1/6628764#supplementary-data

### PR DESCRIPTION
**Pull request by ClementCaporal**
Hello,

Here are some new ressources!

added:

- https://zenodo.org/records/10815329
- https://github.com/BiAPoL/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis/
- https://github.com/ScaDS/BIDS-lecture-2024
- https://github.com/computational-cell-analytics/dl-for-micro
- https://zoccoler.github.io/QM_Course_Bio_Image_Analysis_with_napari_202 closes #61
- https://bethac07.github.io/qi_2024_analysis_lab_manual/intro.html
- https://plantsandpython.github.io/PlantsAndPython/00_Opening_page.html closes #62
- https://zenodo.org/records/10679054 closes #60
- https://zenodo.org/records/10654775 closes #60
- https://zenodo.org/records/10008465 closes #55
- https://www.youtube.com/@RDM4mic closes #51
- https://www.youtube.com/watch?v=A4po9z61TME closes #53 (directly in the DigitalSreeni urls)

### CODE REVIEW

File: resources/materials.yml [hunk 1]
```diff
@@ -1235,3 +1235,113 @@ resources:
...
   type: slide
   url: https://zenodo.org/records/10083555
+
+- authors: Robert Haase
+  license: cc-by-4.0
+  name: Bio-image Analysis with the Help of Large Language Models
+  description: Large Language Models (LLMs) change the way how we use computers. This also has impact on the bio-image analysis community. We can
+  generate code that analyzes biomedical image data if we ask the right prompts. This talk outlines introduces basic principles, explains prompt
+  engineering and how to apply it to bio-image analysis. We also compare how different LLM vendors perform on code generation tasks and which challenges
+  are ahead for the bio-image analysis community.
```
Comment (haesleinhuepf) [inline resources/materials.yml:1245]:
```suggestion
  description: Large Language Models (LLMs) change the way how we use computers. This also has impact on the bio-image analysis community. We can generate code that analyzes biomedical image data if we ask the right prompts. This talk outlines introduces basic principles, explains prompt engineering and how to apply it to bio-image analysis. We also compare how different LLM vendors perform on code generation tasks and which challenges are ahead for the bio-image analysis community.
```

File: resources/materials.yml [hunk 2]
```diff
@@ -1235,3 +1235,110 @@ resources:
...
+  - python
+  type:
+  - notebook
+  url: https://github.com/ScaDS/BIDS-lecture-2024
+
+- authors: Costantin Pape, computational-cell-analytics
+  license: MIT License
+  name: Introduction to Deep Learning for Microscopy
+  description: This course consists of lectures and exercises that teach the background of deep learning for image analysis
+  and show applications to classification and segmentation analysis problems.
```
Comment (haesleinhuepf) [inline resources/materials.yml:1279]:
```suggestion
  description: This course consists of lectures and exercises that teach the background of deep learning for image analysis and show applications to classification and segmentation analysis problems.
```

File: resources/materials.yml [hunk 3]
```diff
@@ -1235,3 +1235,109 @@ resources:
...
+  type:
+  - notebook
+  url: https://github.com/computational-cell-analytics/dl-for-micro
+
+- authors: Marcelo Leomil Zoccoler
+  license: cc-by-4.0
+  name: QM Course Lectures on Bio-Image Analysis with napari 2024
+  description: In these lectures, we will explore ways to analyze microscopy images with Python and visualize them with napari, an nD viewer open-source software.
+  The analysis will be done in Python mostly using the scikit-image, pyclesperanto and apoc libraries, via Jupyter notebooks. We will also explore some napari plugins
+  as an interactive and convenient alternative way of performing these analysis, especially the napari-assistant, napari-apoc and napari-flim-phasor-plotter plugins.
```
Comment (haesleinhuepf) [inline resources/materials.yml:1293]:
```suggestion
  description: In these lectures, we will explore ways to analyze microscopy images with Python and visualize them with napari, an nD viewer open-source software. The analysis will be done in Python mostly using the scikit-image, pyclesperanto and apoc libraries, via Jupyter notebooks. We will also explore some napari plugins as an interactive and convenient alternative way of performing these analysis, especially the napari-assistant, napari-apoc and napari-flim-phasor-plotter plugins.
```

File: resources/materials.yml [hunk 4]
```diff
@@ -1235,3 +1235,107 @@ resources:
...
+  - python
+  type:
+  - notebook
+  url: https://bethac07.github.io/qi_2024_analysis_lab_manual/intro.html
+
+- authors: Elnaz Fazeli 
+  license: cc-by-4.0
+  name: My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud
+  description: In these slides I introducemy journey through teaching bioimage analysis courses in different formats, from in person courses to online material.
+  I have an overview of different training formats and comparing these for different audiences.
```
Comment (haesleinhuepf) [inline resources/materials.yml:1314]:
```suggestion
  description: In these slides I introducemy journey through teaching bioimage analysis courses in different formats, from in person courses to online material. I have an overview of different training formats and comparing these for different audiences. 
```

File: resources/materials.yml [hunk 5]
```diff
@@ -1235,3 +1235,106 @@ resources:
...
+  type:
+  - slides
+  url: https://zenodo.org/records/10679054
+
+- authors: Robert Haase
+  license: cc-by-4.0
+  name: Cultivating Open Training
+  description: In these slides introduce current challenges and potential solutions for openly sharing training materials, softly focusing on bio-image analysis.
+  In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare.
+  Using the CC-BY license and publicly acessible repositories are proposed to fill this gap.
```
Comment (haesleinhuepf) [inline resources/materials.yml:1325]:
```suggestion
  description: In these slides introduce current challenges and potential solutions for openly sharing training materials, softly focusing on bio-image analysis. In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare. Using the CC-BY license and publicly acessible repositories are proposed to fill this gap.
```

**Review by haesleinhuepf — APPROVED** [review]
Awesome, thanks @ClementCaporal ! Just FYI: I had to remove some line breaks and add some quotes around square brackets to make the book compile.

**Comment by ClementCaporal** [comment]
Oups, Sorry I didn't check the book compile I will check next time!

Thanks

**Pull request merged**